frontend/src/store/modules/maps.js

```diff
@@ -38,6 +38,7 @@ const data = {
   backgroundColor: BG_COLORS.light,
   loading: true,
   loadingElement: false,
+  svgReactionsIdList: null,
```

Comment:
Sure. I agree it is better to initialize an array as [] instead of null, in case e.g. `array.length` is called.
However, It a bit weird that I did not see any Vue error with `/explore/Human-GEM/map-viewer/acylglycerides_metabolism?dim=2d`